config/locales/user_mailer/en.yml

```diff
@@ -44,6 +44,17 @@ en:
         %{help_link} or %{contact_link}.
       intro: The email address for your %{app} account has been changed.
     help_link_text: Help Center
+    letter_expired:
+      help: ''
+      info: The letter we sent to your address has a code that expired and can no
+        longer be used to complete the verification process.  To verify your identity
+        you will need to sign into %{link} and choose to send another letter.
+      subject: The letter we sent to your address has a code that has expired
+    letter_reminder:
+      help: ''
+      info: When you receive the letter it will contain a code.  Verify your identity
```

Comment:
`info: The letter you are about to receive will contain a confirmation code that helps us verify your address. You can complete the identity verification process by signing into {%link} and entering the confirmation code`